packages/gatsby-source-wordpress/package.json

```diff
@@ -8,6 +8,13 @@
     "lodash": "^4.17.4",
     "qs": "^6.4.0"
   },
+  "devDependencies": {
```

Comment:
Sorry, done. How do you run `npm run watch` using the config from the base of the repo?